spec/factories/group_sets_factory.rb

```diff
@@ -1,8 +0,0 @@
-# Read about factories at https://github.com/thoughtbot/factory_girl
```

Comment:
It would be good to keep the old spec tests until they are switched to minitest.